csrc/scheduler/cache_policy_refiner.cpp

```diff
@@ -0,0 +1,128 @@
+#include <fusion.h>
+#include <ir/base_nodes.h>
+#include <ir/internal_nodes.h>
+#include <ir/utils.h>
+#include <scheduler/cache_policy_refiner.h>
+#include <scheduler/debug_utils.h>
+
+namespace nvfuser {
+
+// Returns whether a pointwise expression `expr` expands its input operand
+// `in_tv`.
+static bool pointwiseExpands(const Expr* expr, const TensorView* in_tv) {
```

Comment:
nit: we generally use anonymous namespaces for file-scope functions.

Reply:
Is there a strong reason for that? I used to use anonymous namespaces but then found it's too hard for me to tell whether a function is in an anonymous namespace, so I switched to `static` instead.